lib/auth/grpcserver.go

```diff
@@ -1866,6 +1867,31 @@ func (g *GRPCServer) IsMFARequired(ctx context.Context, req *proto.IsMFARequired
 	return resp, nil
 }
 
+func (g *GRPCServer) GetPAMConfig(ctx context.Context, _ *emptypb.Empty) (*types.PAMConfigV3, error) {
+	actx, err := g.authenticate(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+	resp, err := actx.GetPAMConfig(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+	concrete := resp.(*types.PAMConfigV3)
```

Comment:
panic unless a type-assertion is used. I believe the rest of the code is paranoid about that.